src/app/services/services.module.ts

```diff
@@ -163,7 +163,11 @@ export function initFactory(): Function {
         htmlEl.classList.add('locale_' + i18nService.translationLocale);
         let theme = await storageService.get<string>(ConstantsService.themeKey);
         if (theme == null) {
-            theme = 'light';
+            theme = await platformUtilsService.getDefaultSystemTheme();
+            platformUtilsService.onDefaultSystemThemeChange(sysTheme => {
+                htmlEl.classList.remove('theme_light', 'theme_dark');
```

Comment:
There's a bit of a bug at the moment where this event handler is only registered when the app is started based on the theme at that point in time, so if you change the theme later it will not activate or deactivate the event handler.

To avoid this, I recommend always registering the event handler here, and then checking whether `theme == null` inside the callback.